build_tools/python/e2e_test_artifacts/cmake_generator/iree_generator.py

```diff
@@ -0,0 +1,218 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Generates CMake rules to build IREE artifacts."""
```

Comment:
`iree_generator` reads a bit weird. I _think_ the goal is to generate the cmake rules for the IREE modules, not generating `iree`. Maybe rename the file?

Reply:
Renamed to `iree_rule_generator`